new/components/vcloud/update-vcloud-vm.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash -e
+
+uname -a
+
+# update system
+yum update --skip-broken -y
```

Comment:
gcc - required for lxml(https://github.com/vmware/pyvcloud/blob/master/requirements.txt#L2) package.